.rspec_status

```diff
@@ -0,0 +1,4 @@
+example_id                                             | status | run_time        |
```

Comment:
Do you need this.

Reply:
yea to check status of test when compiling but i will remove in the next commit